### PR TITLE
New package: odrive-0.2.0

### PR DESCRIPTION
**Pull request by jaredycalves**

### CODE REVIEW

File: srcpkgs/odrive/template [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+version=0.2.0
+revision=1
+create_wrksrc=yes
+only_for_archs="x86_64"
+short_desc="Google Drive GUI for linux"
+maintainer="Jaredy <jaredycalves@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/liberodark/ODrive"
+distfiles="https://github.com/liberodark/ODrive/releases/download/${version}/odrive_${version}_amd64.deb"
+checksum="31fac62a0b9b8d840708e9243f72fa260ca66e716f1af1b4844fa4f0876d57d7"
```
Comment (maxice8) [inline srcpkgs/odrive/template]: checksum doesn't need to be quoted

File: srcpkgs/odrive/template [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+pkgname=odrive
+version=0.2.0
+revision=1
+create_wrksrc=yes
+only_for_archs="x86_64"
+short_desc="Google Drive GUI for linux"
+maintainer="Jaredy <jaredycalves@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/liberodark/ODrive"
+distfiles="https://github.com/liberodark/ODrive/releases/download/${version}/odrive_${version}_amd64.deb"
```
Comment (maxice8) [inline srcpkgs/odrive/template]: get from source code and compile it if it's free software

File: srcpkgs/odrive/template (hunk 1 above)
Comment (jaredycalves) [reply srcpkgs/odrive/template]: Removed

File: srcpkgs/odrive/template (hunk 2 above)
Comment (jaredycalves) [reply srcpkgs/odrive/template]: Done

File: srcpkgs/odrive/template [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# Template file for 'odrive'
+pkgname=odrive
+version=0.2.0
+revision=1
+only_for_archs="x86_64"
+makedepends=nodejs
```
Comment (maxice8) [inline srcpkgs/odrive/template]: should be quoted

File: srcpkgs/odrive/template [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+do_build() {
+	cd ${wrksrc}
+	npm install
+	npm run release-linux dir
+}
+
+do_install() {
+	vmkdir ${destdir}/opt/ODrive
```
Comment (maxice8) [inline srcpkgs/odrive/template]: doesn't exist, you're looking for DESTDIR, which is added implicty on relative paths.

opt/ODrive

File: srcpkgs/odrive/template [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+	vmkdir ${destdir}/usr/share/applications
+
+	vcopy ${wrksrc}/dist/linux-unpacked/* ${destdir}/opt/ODrive
```
Comment (maxice8) [inline srcpkgs/odrive/template]: wrksrc is implicit, destdir doesn't exist

File: srcpkgs/odrive/template [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+short_desc="Google Drive GUI for linux"
+maintainer="Jaredy <jaredycalves@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/liberodark/ODrive"
+distfiles="https://github.com/liberodark/ODrive/archive/${version}.tar.gz"
+checksum=5e239502937711d6609eacccdcc332a2f0c8d92412d7461439ce00bf180dfaf2
+nopie=yes
+
+do_extract() {
+	tar -xvf ${XBPS_SRCDISTDIR}/${pkgname}-${version}/${version}.tar.gz --strip 1
```
Comment (maxice8) [inline srcpkgs/odrive/template]: you problably want the fetch build_style

File: srcpkgs/odrive/template [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+do_build() {
+	cd ${wrksrc}
```
Comment (maxice8) [inline srcpkgs/odrive/template]: do_build starts at wrksrc, is this copied from the AUR ?

**Pull request closed without merging**